src/components/dashboard-settings/sync-apps/sync-apps.directive.js.coffee

```diff
@@ -125,30 +119,63 @@ module.directive('dashboardSettingSyncApps', ($templateCache, $log, $http, $filt
             scope.isSyncing = false
         )
 
-      scope.formatStatus = (connector) ->
+      scope.formatStatus = (connector, modalDisplay=false) ->
         return unless connector
         name = connector.name
         status = ""
 
         if connector.last_sync_date
-          date = $filter('date')(connector.last_sync_date, "yyyy-MM-dd 'at' h:mma", getOffset())
-        
+          date = $filter('date')(connector.last_sync_date, "h:mma, yyyy-MM-dd", getOffset())
+
           switch connector.status
-            when 'SUCCESS' then status = "Last sync: #{date}"
-            when 'FAILED' then status = "Last sync failed: #{date}"
-            when 'DISCONNECTED' then status = "Disconnected - previous sync was: #{date}"
+            when 'SUCCESS'
+              status = if modalDisplay then "Synced at #{date}" else "is synced - #{date}"
+            when 'FAILED'
+              status = if modalDisplay then "Previous sync #{date}" else "is not synced - previous sync #{date}"
+            when 'DISCONNECTED'
+              status = if modalDisplay then "Previous sync #{date}" else "is disconnected - previous sync #{date}"
             # "RUNNING" case should imply isSyncing==true...
 
         else
           switch connector.status
-            when 'FAILED' then status = "Sync failed"
-            when 'DISCONNECTED' then status = "Sync failed - Disconnected"
-            when 'NOT SYNCED' then status = "Not synced yet"
+            when 'FAILED', 'NOT SYNCED'
```

Comment:
Why would statuses 'FAILED' or 'DISCONNECTED' display the message "never synced" ?